builds.yml

```diff
@@ -98,7 +98,7 @@ buildTypes:
       - INFURA_ENV_KEY_REF: INFURA_EXPERIMENTAL_PROJECT_ID
       - SEGMENT_WRITE_KEY_REF: SEGMENT_EXPERIMENTAL_WRITE_KEY
       - ALLOW_LOCAL_SNAPS: false
-      - IS_SIDEPANEL: true
+      - IS_SIDEPANEL: false
```

Comment:
Nit: The permission is still in the experimental manifest, so we can leave it on here